_sass/_header.scss

```diff
@@ -28,6 +41,33 @@ header {
       // MS Edge doesn't like unbounded SVG images.
       // This value was derived empirically using Chrome 69.
       max-width: 501px;
+
+      @media (max-width: 960px){
+        & {
+          height: 38px;
```

Comment:
Where does this value come from? Just curious

Reply:
Just experimented with different sizes and thought this fits the navbar best.